deploy.sh

```diff
@@ -6,3 +6,6 @@ git checkout $1
 
 git submodule init
 git submodule update
+
+rm -r templates_c/
+mkdir templates_c/
```

Comment:
I don't think this is strictly necessary since Smarty will create this if the directory doesn't exist, but nbd either way